data/alibaba-ads

```diff
@@ -28,6 +28,7 @@ atanx.alicdn.com @ads
 atanx2.alicdn.com @ads
 c-adash.m.taobao.com @ads
 cdn0.mobmore.com @ads
+cdn-mum.alibabachengdun.com @ads
```

Comment:
According to the ASCII code, `cdn-mum.alibabachengdun.com` should be put in front of `cdn0.mobmore.com`.